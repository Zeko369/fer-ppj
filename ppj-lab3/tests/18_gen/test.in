<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 z
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 1078
      <T_lista>
       $
     <E_lista>
      OP_PLUS 1 +
      <E>
       <T>
        <P>
         BROJ 1 545
        <T_lista>
         OP_PUTA 1 *
         <T>
          <P>
           BROJ 1 689
          <T_lista>
           $
       <E_lista>
        $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 z
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 501
       <T_lista>
        OP_PUTA 2 *
        <T>
         <P>
          BROJ 2 207
         <T_lista>
          $
      <E_lista>
       OP_MINUS 2 -
       <E>
        <T>
         <P>
          BROJ 2 997
         <T_lista>
          OP_PUTA 2 *
          <T>
           <P>
            OP_PLUS 2 +
            <P>
             IDN 2 z
           <T_lista>
            $
        <E_lista>
         $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 g
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         BROJ 3 383
        <T_lista>
         $
       <E_lista>
        OP_PLUS 3 +
        <E>
         <T>
          <P>
           L_ZAGRADA 3 (
           <E>
            <T>
             <P>
              L_ZAGRADA 3 (
              <E>
               <T>
                <P>
                 IDN 3 z
                <T_lista>
                 OP_PUTA 3 *
                 <T>
                  <P>
                   L_ZAGRADA 3 (
                   <E>
                    <T>
                     <P>
                      L_ZAGRADA 3 (
                      <E>
                       <T>
                        <P>
                         BROJ 3 142
                        <T_lista>
                         $
                       <E_lista>
                        $
                      D_ZAGRADA 3 )
                     <T_lista>
                      $
                    <E_lista>
                     $
                   D_ZAGRADA 3 )
                  <T_lista>
                   $
               <E_lista>
                $
              D_ZAGRADA 3 )
             <T_lista>
              $
            <E_lista>
             $
           D_ZAGRADA 3 )
          <T_lista>
           $
         <E_lista>
          $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 4 m
       OP_PRIDRUZI 4 =
       <E>
        <T>
         <P>
          BROJ 4 1087
         <T_lista>
          OP_PUTA 4 *
          <T>
           <P>
            OP_PLUS 4 +
            <P>
             BROJ 4 221
           <T_lista>
            $
        <E_lista>
         OP_MINUS 4 -
         <E>
          <T>
           <P>
            OP_PLUS 4 +
            <P>
             OP_MINUS 4 -
             <P>
              BROJ 4 608
           <T_lista>
            $
          <E_lista>
           OP_PLUS 4 +
           <E>
            <T>
             <P>
              IDN 4 g
             <T_lista>
              $
            <E_lista>
             $
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 5 r
        OP_PRIDRUZI 5 =
        <E>
         <T>
          <P>
           IDN 5 m
          <T_lista>
           $
         <E_lista>
          $
      <lista_naredbi>
       <naredba>
        <naredba_pridruzivanja>
         IDN 6 t
         OP_PRIDRUZI 6 =
         <E>
          <T>
           <P>
            BROJ 6 700
           <T_lista>
            $
          <E_lista>
           OP_MINUS 6 -
           <E>
            <T>
             <P>
              BROJ 6 875
             <T_lista>
              $
            <E_lista>
             $
       <lista_naredbi>
        <naredba>
         <za_petlja>
          KR_ZA 7 za
          IDN 7 g
          KR_OD 7 od
          <E>
           <T>
            <P>
             BROJ 7 927
            <T_lista>
             OP_PUTA 7 *
             <T>
              <P>
               L_ZAGRADA 7 (
               <E>
                <T>
                 <P>
                  BROJ 7 390
                 <T_lista>
                  $
                <E_lista>
                 $
               D_ZAGRADA 7 )
              <T_lista>
               $
           <E_lista>
            OP_MINUS 7 -
            <E>
             <T>
              <P>
               BROJ 7 714
              <T_lista>
               $
             <E_lista>
              $
          KR_DO 7 do
          <E>
           <T>
            <P>
             BROJ 7 922
            <T_lista>
             $
           <E_lista>
            OP_PLUS 7 +
            <E>
             <T>
              <P>
               IDN 7 g
              <T_lista>
               $
             <E_lista>
              OP_PLUS 7 +
              <E>
               <T>
                <P>
                 OP_MINUS 7 -
                 <P>
                  IDN 7 r
                <T_lista>
                 $
               <E_lista>
                OP_MINUS 7 -
                <E>
                 <T>
                  <P>
                   BROJ 7 625
                  <T_lista>
                   $
                 <E_lista>
                  $
          <lista_naredbi>
           <naredba>
            <naredba_pridruzivanja>
             IDN 8 u
             OP_PRIDRUZI 8 =
             <E>
              <T>
               <P>
                OP_PLUS 8 +
                <P>
                 OP_MINUS 8 -
                 <P>
                  BROJ 8 587
               <T_lista>
                $
              <E_lista>
               $
           <lista_naredbi>
            <naredba>
             <naredba_pridruzivanja>
              IDN 9 u
              OP_PRIDRUZI 9 =
              <E>
               <T>
                <P>
                 L_ZAGRADA 9 (
                 <E>
                  <T>
                   <P>
                    BROJ 9 344
                   <T_lista>
                    $
                  <E_lista>
                   OP_MINUS 9 -
                   <E>
                    <T>
                     <P>
                      BROJ 9 847
                     <T_lista>
                      $
                    <E_lista>
                     OP_MINUS 9 -
                     <E>
                      <T>
                       <P>
                        BROJ 9 39
                       <T_lista>
                        $
                      <E_lista>
                       $
                 D_ZAGRADA 9 )
                <T_lista>
                 OP_PUTA 9 *
                 <T>
                  <P>
                   OP_MINUS 9 -
                   <P>
                    BROJ 9 978
                  <T_lista>
                   $
               <E_lista>
                $
            <lista_naredbi>
             <naredba>
              <naredba_pridruzivanja>
               IDN 10 d
               OP_PRIDRUZI 10 =
               <E>
                <T>
                 <P>
                  BROJ 10 218
                 <T_lista>
                  $
                <E_lista>
                 $
             <lista_naredbi>
              <naredba>
               <za_petlja>
                KR_ZA 11 za
                IDN 11 m
                KR_OD 11 od
                <E>
                 <T>
                  <P>
                   BROJ 11 465
                  <T_lista>
                   OP_PUTA 11 *
                   <T>
                    <P>
                     OP_PLUS 11 +
                     <P>
                      OP_PLUS 11 +
                      <P>
                       L_ZAGRADA 11 (
                       <E>
                        <T>
                         <P>
                          IDN 11 t
                         <T_lista>
                          $
                        <E_lista>
                         $
                       D_ZAGRADA 11 )
                    <T_lista>
                     $
                 <E_lista>
                  OP_PLUS 11 +
                  <E>
                   <T>
                    <P>
                     BROJ 11 910
                    <T_lista>
                     $
                   <E_lista>
                    $
                KR_DO 11 do
                <E>
                 <T>
                  <P>
                   BROJ 11 826
                  <T_lista>
                   $
                 <E_lista>
                  OP_PLUS 11 +
                  <E>
                   <T>
                    <P>
                     BROJ 11 444
                    <T_lista>
                     OP_PUTA 11 *
                     <T>
                      <P>
                       L_ZAGRADA 11 (
                       <E>
                        <T>
                         <P>
                          BROJ 11 1052
                         <T_lista>
                          OP_PUTA 11 *
                          <T>
                           <P>
                            IDN 11 d
                           <T_lista>
                            $
                        <E_lista>
                         $
                       D_ZAGRADA 11 )
                      <T_lista>
                       $
                   <E_lista>
                    $
                <lista_naredbi>
                 <naredba>
                  <naredba_pridruzivanja>
                   IDN 12 u
                   OP_PRIDRUZI 12 =
                   <E>
                    <T>
                     <P>
                      IDN 12 t
                     <T_lista>
                      $
                    <E_lista>
                     OP_MINUS 12 -
                     <E>
                      <T>
                       <P>
                        BROJ 12 650
                       <T_lista>
                        $
                      <E_lista>
                       OP_MINUS 12 -
                       <E>
                        <T>
                         <P>
                          IDN 12 g
                         <T_lista>
                          $
                        <E_lista>
                         $
                 <lista_naredbi>
                  <naredba>
                   <naredba_pridruzivanja>
                    IDN 13 m
                    OP_PRIDRUZI 13 =
                    <E>
                     <T>
                      <P>
                       BROJ 13 564
                      <T_lista>
                       $
                     <E_lista>
                      $
                  <lista_naredbi>
                   $
                KR_AZ 14 az
              <lista_naredbi>
               $
          KR_AZ 15 az
        <lista_naredbi>
         <naredba>
          <naredba_pridruzivanja>
           IDN 16 x
           OP_PRIDRUZI 16 =
           <E>
            <T>
             <P>
              BROJ 16 80
             <T_lista>
              $
            <E_lista>
             $
         <lista_naredbi>
          <naredba>
           <naredba_pridruzivanja>
            IDN 17 x
            OP_PRIDRUZI 17 =
            <E>
             <T>
              <P>
               BROJ 17 64
              <T_lista>
               $
             <E_lista>
              OP_MINUS 17 -
              <E>
               <T>
                <P>
                 IDN 17 r
                <T_lista>
                 $
               <E_lista>
                OP_MINUS 17 -
                <E>
                 <T>
                  <P>
                   IDN 17 z
                  <T_lista>
                   $
                 <E_lista>
                  $
          <lista_naredbi>
           <naredba>
            <naredba_pridruzivanja>
             IDN 18 a
             OP_PRIDRUZI 18 =
             <E>
              <T>
               <P>
                L_ZAGRADA 18 (
                <E>
                 <T>
                  <P>
                   BROJ 18 490
                  <T_lista>
                   $
                 <E_lista>
                  OP_PLUS 18 +
                  <E>
                   <T>
                    <P>
                     BROJ 18 118
                    <T_lista>
                     $
                   <E_lista>
                    $
                D_ZAGRADA 18 )
               <T_lista>
                OP_PUTA 18 *
                <T>
                 <P>
                  IDN 18 m
                 <T_lista>
                  $
              <E_lista>
               OP_MINUS 18 -
               <E>
                <T>
                 <P>
                  BROJ 18 968
                 <T_lista>
                  $
                <E_lista>
                 $
           <lista_naredbi>
            <naredba>
             <naredba_pridruzivanja>
              IDN 19 g
              OP_PRIDRUZI 19 =
              <E>
               <T>
                <P>
                 IDN 19 x
                <T_lista>
                 $
               <E_lista>
                OP_PLUS 19 +
                <E>
                 <T>
                  <P>
                   BROJ 19 1098
                  <T_lista>
                   $
                 <E_lista>
                  $
            <lista_naredbi>
             <naredba>
              <naredba_pridruzivanja>
               IDN 20 l
               OP_PRIDRUZI 20 =
               <E>
                <T>
                 <P>
                  BROJ 20 578
                 <T_lista>
                  $
                <E_lista>
                 OP_MINUS 20 -
                 <E>
                  <T>
                   <P>
                    IDN 20 a
                   <T_lista>
                    $
                  <E_lista>
                   OP_PLUS 20 +
                   <E>
                    <T>
                     <P>
                      IDN 20 z
                     <T_lista>
                      $
                    <E_lista>
                     $
             <lista_naredbi>
              <naredba>
               <naredba_pridruzivanja>
                IDN 21 h
                OP_PRIDRUZI 21 =
                <E>
                 <T>
                  <P>
                   BROJ 21 843
                  <T_lista>
                   $
                 <E_lista>
                  $
              <lista_naredbi>
               <naredba>
                <naredba_pridruzivanja>
                 IDN 22 l
                 OP_PRIDRUZI 22 =
                 <E>
                  <T>
                   <P>
                    BROJ 22 895
                   <T_lista>
                    OP_PUTA 22 *
                    <T>
                     <P>
                      IDN 22 l
                     <T_lista>
                      $
                  <E_lista>
                   $
               <lista_naredbi>
                <naredba>
                 <naredba_pridruzivanja>
                  IDN 23 m
                  OP_PRIDRUZI 23 =
                  <E>
                   <T>
                    <P>
                     BROJ 23 1026
                    <T_lista>
                     OP_PUTA 23 *
                     <T>
                      <P>
                       BROJ 23 572
                      <T_lista>
                       OP_PUTA 23 *
                       <T>
                        <P>
                         BROJ 23 279
                        <T_lista>
                         $
                   <E_lista>
                    OP_PLUS 23 +
                    <E>
                     <T>
                      <P>
                       BROJ 23 1083
                      <T_lista>
                       OP_PUTA 23 *
                       <T>
                        <P>
                         BROJ 23 408
                        <T_lista>
                         $
                     <E_lista>
                      $
                <lista_naredbi>
                 <naredba>
                  <naredba_pridruzivanja>
                   IDN 24 x
                   OP_PRIDRUZI 24 =
                   <E>
                    <T>
                     <P>
                      BROJ 24 84
                     <T_lista>
                      $
                    <E_lista>
                     $
                 <lista_naredbi>
                  <naredba>
                   <za_petlja>
                    KR_ZA 25 za
                    IDN 25 m
                    KR_OD 25 od
                    <E>
                     <T>
                      <P>
                       IDN 25 h
                      <T_lista>
                       OP_PUTA 25 *
                       <T>
                        <P>
                         L_ZAGRADA 25 (
                         <E>
                          <T>
                           <P>
                            IDN 25 g
                           <T_lista>
                            OP_PUTA 25 *
                            <T>
                             <P>
                              BROJ 25 665
                             <T_lista>
                              $
                          <E_lista>
                           $
                         D_ZAGRADA 25 )
                        <T_lista>
                         $
                     <E_lista>
                      OP_MINUS 25 -
                      <E>
                       <T>
                        <P>
                         IDN 25 l
                        <T_lista>
                         $
                       <E_lista>
                        OP_MINUS 25 -
                        <E>
                         <T>
                          <P>
                           BROJ 25 530
                          <T_lista>
                           $
                         <E_lista>
                          $
                    KR_DO 25 do
                    <E>
                     <T>
                      <P>
                       IDN 25 z
                      <T_lista>
                       $
                     <E_lista>
                      $
                    <lista_naredbi>
                     <naredba>
                      <naredba_pridruzivanja>
                       IDN 26 c
                       OP_PRIDRUZI 26 =
                       <E>
                        <T>
                         <P>
                          BROJ 26 1098
                         <T_lista>
                          OP_PUTA 26 *
                          <T>
                           <P>
                            BROJ 26 862
                           <T_lista>
                            $
                        <E_lista>
                         $
                     <lista_naredbi>
                      <naredba>
                       <naredba_pridruzivanja>
                        IDN 27 x
                        OP_PRIDRUZI 27 =
                        <E>
                         <T>
                          <P>
                           BROJ 27 188
                          <T_lista>
                           $
                         <E_lista>
                          OP_PLUS 27 +
                          <E>
                           <T>
                            <P>
                             BROJ 27 863
                            <T_lista>
                             $
                           <E_lista>
                            $
                      <lista_naredbi>
                       $
                    KR_AZ 28 az
                  <lista_naredbi>
                   $
