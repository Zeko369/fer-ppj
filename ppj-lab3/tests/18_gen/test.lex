IDN 1 z
OP_PRIDRUZI 1 =
BROJ 1 1078
OP_PLUS 1 +
BROJ 1 545
OP_PUTA 1 *
BROJ 1 689
IDN 2 z
OP_PRIDRUZI 2 =
BROJ 2 501
OP_PUTA 2 *
BROJ 2 207
OP_MINUS 2 -
BROJ 2 997
OP_PUTA 2 *
OP_PLUS 2 +
IDN 2 z
IDN 3 g
OP_PRIDRUZI 3 =
BROJ 3 383
OP_PLUS 3 +
L_ZAGRADA 3 (
L_ZAGRADA 3 (
IDN 3 z
OP_PUTA 3 *
L_ZAGRADA 3 (
L_ZAGRADA 3 (
BROJ 3 142
D_ZAGRADA 3 )
D_ZAGRADA 3 )
D_ZAGRADA 3 )
D_ZAGRADA 3 )
IDN 4 m
OP_PRIDRUZI 4 =
BROJ 4 1087
OP_PUTA 4 *
OP_PLUS 4 +
BROJ 4 221
OP_MINUS 4 -
OP_PLUS 4 +
OP_MINUS 4 -
BROJ 4 608
OP_PLUS 4 +
IDN 4 g
IDN 5 r
OP_PRIDRUZI 5 =
IDN 5 m
IDN 6 t
OP_PRIDRUZI 6 =
BROJ 6 700
OP_MINUS 6 -
BROJ 6 875
KR_ZA 7 za
IDN 7 g
KR_OD 7 od
BROJ 7 927
OP_PUTA 7 *
L_ZAGRADA 7 (
BROJ 7 390
D_ZAGRADA 7 )
OP_MINUS 7 -
BROJ 7 714
KR_DO 7 do
BROJ 7 922
OP_PLUS 7 +
IDN 7 g
OP_PLUS 7 +
OP_MINUS 7 -
IDN 7 r
OP_MINUS 7 -
BROJ 7 625
IDN 8 u
OP_PRIDRUZI 8 =
OP_PLUS 8 +
OP_MINUS 8 -
BROJ 8 587
IDN 9 u
OP_PRIDRUZI 9 =
L_ZAGRADA 9 (
BROJ 9 344
OP_MINUS 9 -
BROJ 9 847
OP_MINUS 9 -
BROJ 9 39
D_ZAGRADA 9 )
OP_PUTA 9 *
OP_MINUS 9 -
BROJ 9 978
IDN 10 d
OP_PRIDRUZI 10 =
BROJ 10 218
KR_ZA 11 za
IDN 11 m
KR_OD 11 od
BROJ 11 465
OP_PUTA 11 *
OP_PLUS 11 +
OP_PLUS 11 +
L_ZAGRADA 11 (
IDN 11 t
D_ZAGRADA 11 )
OP_PLUS 11 +
BROJ 11 910
KR_DO 11 do
BROJ 11 826
OP_PLUS 11 +
BROJ 11 444
OP_PUTA 11 *
L_ZAGRADA 11 (
BROJ 11 1052
OP_PUTA 11 *
IDN 11 d
D_ZAGRADA 11 )
IDN 12 u
OP_PRIDRUZI 12 =
IDN 12 t
OP_MINUS 12 -
BROJ 12 650
OP_MINUS 12 -
IDN 12 g
IDN 13 m
OP_PRIDRUZI 13 =
BROJ 13 564
KR_AZ 14 az
KR_AZ 15 az
IDN 16 x
OP_PRIDRUZI 16 =
BROJ 16 80
IDN 17 x
OP_PRIDRUZI 17 =
BROJ 17 64
OP_MINUS 17 -
IDN 17 r
OP_MINUS 17 -
IDN 17 z
IDN 18 a
OP_PRIDRUZI 18 =
L_ZAGRADA 18 (
BROJ 18 490
OP_PLUS 18 +
BROJ 18 118
D_ZAGRADA 18 )
OP_PUTA 18 *
IDN 18 m
OP_MINUS 18 -
BROJ 18 968
IDN 19 g
OP_PRIDRUZI 19 =
IDN 19 x
OP_PLUS 19 +
BROJ 19 1098
IDN 20 l
OP_PRIDRUZI 20 =
BROJ 20 578
OP_MINUS 20 -
IDN 20 a
OP_PLUS 20 +
IDN 20 z
IDN 21 h
OP_PRIDRUZI 21 =
BROJ 21 843
IDN 22 l
OP_PRIDRUZI 22 =
BROJ 22 895
OP_PUTA 22 *
IDN 22 l
IDN 23 m
OP_PRIDRUZI 23 =
BROJ 23 1026
OP_PUTA 23 *
BROJ 23 572
OP_PUTA 23 *
BROJ 23 279
OP_PLUS 23 +
BROJ 23 1083
OP_PUTA 23 *
BROJ 23 408
IDN 24 x
OP_PRIDRUZI 24 =
BROJ 24 84
KR_ZA 25 za
IDN 25 m
KR_OD 25 od
IDN 25 h
OP_PUTA 25 *
L_ZAGRADA 25 (
IDN 25 g
OP_PUTA 25 *
BROJ 25 665
D_ZAGRADA 25 )
OP_MINUS 25 -
IDN 25 l
OP_MINUS 25 -
BROJ 25 530
KR_DO 25 do
IDN 25 z
IDN 26 c
OP_PRIDRUZI 26 =
BROJ 26 1098
OP_PUTA 26 *
BROJ 26 862
IDN 27 x
OP_PRIDRUZI 27 =
BROJ 27 188
OP_PLUS 27 +
BROJ 27 863
KR_AZ 28 az
