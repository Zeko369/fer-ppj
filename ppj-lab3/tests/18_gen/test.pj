z = 1078 + 545*689
z = 501*207 - 997*+z
g = 383 + ((z*((142))))
m = 1087*+221 - +-608 + g
r = m
t = 700 - 875
za g od 927*(390) - 714 do 922 + g + -r - 625
  u = +-587
  u = (344 - 847 - 39)*-978
  d = 218
  za m od 465*++(t) + 910 do 826 + 444*(1052*d)
    u = t - 650 - g
    m = 564
  az
az
x = 80
x = 64 - r - z
a = (490 + 118)*m - 968
g = x + 1098
l = 578 - a + z
h = 843
l = 895*l
m = 1026*572*279 + 1083*408
x = 84
za m od h*(g*665) - l - 530 do z
  c = 1098*862
  x = 188 + 863
az
