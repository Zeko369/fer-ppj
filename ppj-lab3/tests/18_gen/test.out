2 1 z
3 1 z
4 3 g
5 4 m
err 7 g
